### PR TITLE
GKE Docs Standardization - New PR 

### LINKED ISSUE #9096: [Docs] Standardizing Supported Platform Documentation - GKE
### Current State
Supported platform documentation for GKE is not in an organized order similar to EKS

### Desired State
Standardizing supported platform documentation of GKE Similar to EKS and AKS

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by aboobakersiddiqr63**
**Notes for Reviewers**

This PR fixes #9096 
- Updated gke.md

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ✔] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 0c413b305df52d3295141fce4095cc595a344203 at: https://652d11a09176f8009fdc16d5--meshery-docs-preview.netlify.app

**Comment by saurabh100ni** [comment]
Thanks for raising the PR, let's Discuss this in the Meshery Development Call today,
Please add this as an agenda in the Doc : [Doc](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit)

**Comment by l5io** [comment]
🚀 Preview for commit c44f28363a755059b66d6533fac428478fc141fa at: https://652f5c046676865007e1449c--meshery-docs-preview.netlify.app

File: docs/pages/installation/gke.md [hunk 1]
```diff
@@ -9,56 +9,88 @@ list: include
...
-`mesheryctl system start`
-
-Meshery server supports customizing authentication flow callback URL, which can be configured in the following way
-`MESHERY_SERVER_CALLBACK_URL=https://custom-host mesheryctl system start`
-
-### Manual Configuration (Optional)
-
-If the [Automatic Configuration](#automatic-configuration-recommended) procedure fails or you would like to manually prepare your kubeconfig file to provide Meshery with the necessary access to your managed Kubernetes instance, perform the following actions:
-
-1. Download the [generate_kubeconfig_gke.sh](./generate_kubeconfig_gke.sh) shell script.
```
Comment (leecalcote) [inline docs/pages/installation/gke.md:45]: We're not offering this as an option any longer? No need given `mesheryctl system config`?

Comment (aboobakersiddiqr63) [reply docs/pages/installation/gke.md:45]: if we need manual steps we can add but that manual steps are bit too complex @leecalcote if it is needed i will add it in the commit

Comment (leecalcote) [reply docs/pages/installation/gke.md:45]: I hear that. We might be able to use a 

```yaml
<details>
  <summary>
  </summary>
</details>
```
to keep the info around, but hide the unnecessarily complexity form those readers, who don't need it.

**Pull request closed without merging**